### PR TITLE
feat: add min items value setting for send image

### LINKED ISSUE #91: Only include screenshot on certain clue / drop value
Hi,

First of thanks for this plugin. It's by far the most complete by all others i've used.

Is it possible to show every clue / drop notification but only include a screenshot when the value is above x?

RuneLite_aIOXkBtCNq

Something like: Min Image Value

### PR DESCRIPTION
**Pull request by iProdigy**
Closes #91

### CODE REVIEW

**Review by Felanbird — APPROVED** [review]
Functionality works as intended. 👍

File: src/main/java/dinkplugin/notifiers/ClueNotifier.java [hunk 1]
```diff
@@ -114,13 +113,16 @@ private void handleNotify(Map<Integer, Integer> clueItems) {
...
             );
-            createMessage(DinkPluginConfig::clueSendImage, NotificationBody.builder()
-                .content(notifyMessage)
-                .extra(new ClueNotificationData(clueType, Integer.parseInt(clueCount), itemStacks))
-                .screenshotFile("clueImage.png")
-                .type(NotificationType.CLUE)
-                .embeds(embeds)
-                .build());
+            createMessage(
+                config -> config.clueSendImage() && totalPrice.get() >= config.clueImageMinValue(),
```
Comment (pajlada) [inline src/main/java/dinkplugin/notifiers/ClueNotifier.java:117]: Can we do this calculation outside of `createMessage` to make it more readable?

File: src/main/java/dinkplugin/notifiers/LootNotifier.java [hunk 2]
```diff
@@ -87,18 +86,22 @@ private void handleNotify(Collection<ItemStack> items, String dropper) {
...
             );
-            createMessage(DinkPluginConfig::lootSendImage, NotificationBody.builder()
-                .content(notifyMessage)
-                .embeds(embeds)
-                .extra(new LootNotificationData(serializedItems, dropper))
-                .screenshotFile("lootImage.png")
-                .type(NotificationType.LOOT)
-                .build());
+            createMessage(
+                config -> config.lootSendImage() && totalPrice >= config.lootImageMinValue(),
```
Comment (pajlada) [inline src/main/java/dinkplugin/notifiers/LootNotifier.java:96]: Same as the other min image value checker, perform this outside of the createMessage call for improved readability

File: src/main/java/dinkplugin/DinkPluginConfig.java [hunk 3]
```diff
@@ -620,11 +631,22 @@ default int clueMinValue() {
         return 0;
     }
 
+    @ConfigItem(
+        keyName = "clueImageMinValue",
+        name = "Screenshot Min Value",
+        description = "The minimum value of the items to send a screenshot. Must have 'Send Image' enabled",
```
Comment (pajlada) [inline src/main/java/dinkplugin/DinkPluginConfig.java:637]: Should we specify that this is for the combined value?

**Review by pajlada — CHANGES_REQUESTED** [review]

**Review by pajlada — APPROVED** [review]

**Pull request merged**